src/Sentry.EntityFramework/DbInterceptionIntegration.cs

```diff
@@ -1,17 +1,34 @@
 using System.Data.Entity.Infrastructure.Interception;
+using Sentry.Extensibility;
 using Sentry.Integrations;
 
 namespace Sentry.EntityFramework;
 
 internal class DbInterceptionIntegration : ISdkIntegration
 {
-    private IDbInterceptor? _sqlInterceptor { get; set; }
+    // Internal for testing.
+    internal IDbInterceptor? SqlInterceptor { get; set; }
```

Comment:
perhaps
```
internal IDbInterceptor? SqlInterceptor { get; }
```

Reply:
SqlIntereceptor isn't set at the constructor of this class but on the function `Register`.  So I'm afraid this change isn't possible.